### PR TITLE
Add Bidder query

### PR DESCRIPTION
**Pull request by yuki24**
paired with @erikdstock 
blocked by https://github.com/artsy/metaphysics/pull/1951 and https://github.com/artsy/gravity/pull/12558

In auciton-related code, we have to deal with `bidder_id`s a lot to look up the bidder's `user` and `sale`. This `bidder` query allows us to retrieve both `user` and `sale` in one query, which is very useful.

Let us know if we need to add the same type to v2. 

 * [x] Look to see if we could just add this to v2 without adding it to v1

## Example query:

```graphql
query {
  node(id: "internalID of bidder") {
    ... on Bidder {
      qualifiedForBidding
      user {
        id
        name
        email
      }
      sale {
        id
        name
        liveStartAt
        endAt
        coverImage {
          url
        }
      }
    }
  }
}
```

### CODE REVIEW

**Comment by mzikherman** [comment]
So, any change happening in V1 should happen in V2. That would also include adding new root fields, although we wanted to be more careful/deliberate about that addition in V2.

I think there should have been an ❌ due to the schema drift script part of CI (it would warn that there's a change in V1 that's not in V2)- cc @zephraph .

**Comment by mzikherman** [comment]
In terms of the use case, I sort of understand it, but I'd still like a bit of clarification.

In auction-land, there are places where you have a bidder id, and it's convoluted to get the corresponding user/sale from there (total side-note, in V2, should there be a connection with the node a user or sale, so from a user -> sales, and sale -> users, with the edge fields being the bidder info? feel like that came up at some point), so this lets you get the user and sale clearly?

That..sort of makes sense to me! The thing I'm not fully imagining, is a specific use on the client that has access to the bidder id, but not the corresponding user/sale. Can you link/describe that? If there's just one specific place, for instance, we might adjust that to pass around a user and sale id, instead of the bidder id, or something like that.

Generally speaking (in V2, V1 is wild west), I think we want to be careful and deliberate about what schema choices we make (esp. root fields). If this is just convenience but is possible to query otherwise, we may want to hold off.

**Comment by zephraph** [comment]
Yeeeep, the CI thing is passing incorrectly. I'll have to look into what's going on there.

**Comment by yuki24** [comment]
The `qualified_for_bidding` field is particularly important and it does not exist in either `User` or `Sale` types since it's determined in each sale. Pulse has a few references in e.g. [the `BidderService`](https://github.com/artsy/pulse/blob/master/app/services/bidder_service.rb).

It might be possible to schedule mailer jobs so that it wouldn't have to check the qualification right before sending an email, but that would probably require a fair amount of work in Gravity, Causality, and Pulse.

I have added the same type and query to v2 so it should be good now.

File: src/schema/v2/me/__tests__/create_bidder_mutation.test.js [hunk 1]
```diff
@@ -46,6 +48,7 @@ describe("Bidder mutation", () => {
 
   const context = {
     createBidderLoader: () => Promise.resolve(bidder),
+    saleLoader: sinon.stub().returns(Promise.resolve(sale)),
```
Comment (alloy) [inline src/schema/v2/me/__tests__/create_bidder_mutation.test.js]: Nit: prefer usage of `jest.fn()` over sinon.

**Comment by yuki24** [comment]
Awesome, let's #mergeongreen

**Pull request merged**